nemo/collections/common/data/lhotse/dataloader.py

```diff
@@ -157,6 +158,11 @@ def get_lhotse_dataloader_from_config(
     # 1. Load a manifest as a Lhotse CutSet.
     cuts, is_tarred = read_cutset_from_config(config)
 
+    # Apply channel selector
+    if config.channel_selector is not None:
+        logging.info('Using channel selector %s.', config.channel_selector)
+        cuts = cuts.map(partial(_select_channel, channel_selector=config.channel_selector), apply_fn=None)
```

Comment:
We should leave the default behavior of `.map` here, otherwise it might try to apply this to text-only examples.

```suggestion
        cuts = cuts.map(partial(_select_channel, channel_selector=config.channel_selector))
```